dream-deploy.sh

```diff
@@ -41,7 +43,7 @@ Peculiar options:
   --handle-version-file          Handle version file by overwritting it with a persistent one
 
 Please report bugs in the GitHub issue page or directly on the TES3MP Discord.
-https://github.com/MWMadness/TES3MP-deploy
+https://github.com/DreamWeave-MP/TES3MP-deploy
```

Comment:
This is still referencing TES3MP, you need to change it to dream-deploy

Reply:
Good catch. All instances are now changed to dreamweave, except where we copy the original credits file alongside ours, and the link to GrimKriegor's tes3mp-forge docker.

We'll need a replacement for it under DreamForge when the dockerfile is updated.